tests/bootstrap_tests/test_backups.py

```diff
@@ -61,7 +61,7 @@ def test_cleanup(self):
         )
         with temp_env_var(SGTK_REPO_ROOT=self._core_repo_path):
             config = resolver.resolve_configuration(
-                {"type": "dev", "name": "backup_tests", "path": self._temp_test_path}, self.tk.shotgun
+                {"type": "dev", "name": "backup_tests", "path": self._temp_test_path}, self.mockgun
```

Comment:
It is currently impossible to run a test again a real server. Maybe we should simply rename the `self.mockgun` variable to `self.shotgun`. Then if we ever make it possible to run the tests on a server, we can make sure `self.shotgun` is an actual Shotgun instance? In any case, I would recommend make that sort of sweeping change only when we decide to add the feature.

One thing to keep in mind here is that this change was required because we didn't actually use the pipeline configuration. The only requirement is a connection to Shotgun.